src/nupic/algorithms/SDRClassifier.cpp

```diff
@@ -44,6 +44,11 @@ namespace sdr_classifier {
 
 using namespace std;
 
+
+UInt getClassification( const PDF & data )
```

Comment:
make method of SDRClassifier? so the users don't need to import it separately from the namespace. Classifier is used anyway to obtain the PDF data used here.

Reply:
..done